lib/Inspection/include/Inspection/Transformers.h

```diff
@@ -26,21 +26,35 @@
 #include <chrono>
 #include "Basics/TimeString.h"
 #include "Basics/ErrorCode.h"
+#include "Inspection/Status.h"
+
+#include "date/date.h"
+#include "fmt/core.h"
 
 namespace arangodb::inspection {
 
 struct TimeStampTransformer {
   using SerializedType = std::string;
   using clock = std::chrono::system_clock;
+  static constexpr std::string_view formatString = "%FT%TZ";
   auto toSerialized(clock::time_point source, std::string& target) const
       -> inspection::Status {
-    target = timepointToString(source);
+    target =
+        date::format(formatString.data(), floor<std::chrono::seconds>(source));
```

Comment:
Is this safe? We are now passing a pointer into an `std::string_view`'s data.
AFAIK this pointer is not guaranteed to point to a null-terminated string: https://en.cppreference.com/w/cpp/string/basic_string_view/data
So how can `date::format(pointer, ...)` know where the format string ends?